draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -1152,21 +1217,24 @@ established within the optical network.
 The Ethernet client signal model and the Transparent CBR client
 signal model are used to report all the connectivity services
 provided by the underlay optical tunnels between Ethernet or CBR
-client LTPs, depending on whether the connectivity service is frame-based or transparent. The underlay optical tunnels can be either WDM
+client LTPs, depending on whether the connectivity service is frame-based
+or transparent. The underlay optical tunnels can be either DWDM
 tunnels or, when the optional OTN switching layer is deployed, OTN
 tunnels.
 
-The WDM tunnels can be used to support either Ethernet or CBR client
+The DWDM tunnels can be used to support either Ethernet or CBR client
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
The WDM tunnels can be used to support either Ethernet or CBR client
```